product_packaging_dimension/models/product_packaging.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019-2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    max_weight = fields.Float("Weight (kg)")
+    # lngth IS NOT A TYPO https://github.com/odoo/odoo/issues/41353
+    lngth = fields.Integer("Length (mm)", help="length in millimeters")
+    # Although it feels weird to use Integer in millimeters, we use Int to
+    # override the fields from delivery module instead of defining new ones
+    width = fields.Integer("Width (mm)", help="width in millimeters")
+    height = fields.Integer("Height (mm)", help="height in millimeters")
```

Comment:
The fields are already available in product.packaging
https://github.com/odoo/odoo/blob/14.0/addons/delivery/models/product_packaging.py